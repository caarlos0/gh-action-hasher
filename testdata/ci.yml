name: CI
on: [push]
jobs:
  govulncheck:
    uses: caarlos0/meta/.github/workflows/govulncheck.yml@main
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v5
      - uses: docker/login-action@v2
      - uses: actions/setup-go@v5
      - uses: github/codeql-action/autobuild@v3
      - uses: github/codeql-action/analyze@192325c86100d080feab897ff886c34abd4c83a3
