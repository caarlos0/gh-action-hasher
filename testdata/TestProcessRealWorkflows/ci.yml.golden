name: CI
on: [push]
jobs:
  govulncheck:
    uses: caarlos0/meta/.github/workflows/govulncheck.yml@395f87b2cc4fbb0a99db4decb1b3bbd16bc07cfc
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
      - uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc
      - uses: actions/setup-go@19bb51245e9c80abacb2e91cc42b33fa478b8639
